tests/basic/sharedv4_svc_test.go

```diff
@@ -1307,6 +1309,158 @@ var _ = Describe("{Sharedv4SvcFunctional}", func() {
 	})
 })
 
+// This test runs a ML Workload that predicts Housing rents. It contains one pod that preprocesses model
+// and prepares training data. One pod runs continuously and keeps on retraining the model every 5 mins.
+// The other pods that are deployments run once every 5 mins and read the model, do some prediction and
+// writes their predictions to shared volume on which model is saved. Retraining pod picks this data up
+// and again retrains the model. This is a simple feed-forward neural network model.
+var _ = Describe("{CreateMlWorkloadOnSharedv4Svc}", func() {
+	ns := "ml-workload-ns"
+	appContexts := make([]*scheduler.Context, 0)
+	prereqContexts := make([]*scheduler.Context, 0)
+	taskName := "prepare-ml-workload"
+	totalMlDeps := 5
+	totalMlWorkloads := ReadEnvVariable("NUM_ML_WORKLOADS")
+	if totalMlWorkloads != "" {
+		var err error
+		totalMlDeps, err = strconv.Atoi(totalMlWorkloads)
+		if err != nil {
+			log.Errorf("Failed to convert value %v to int with error: %v", totalMlWorkloads, err)
+			totalMlDeps = 5
+		}
+	}
+	totalRunTime := 5
+	mlWorkloadRuntime := ReadEnvVariable("ML_WORKLOAD_RUNTIME")
+	if mlWorkloadRuntime != "" {
+		var err error
+		totalRunTime, err = strconv.Atoi(mlWorkloadRuntime)
+		if err != nil {
+			log.Errorf("Failed to convert value %v to int with error: %v", mlWorkloadRuntime, err)
+			totalRunTime = 5
+		}
+	}
+	JustBeforeEach(func() {
+		StartTorpedoTest("CreateMlWorkloadOnSharedv4Svc", "Create multiple pods coming and going and trying to edit/read a model on same volume", nil, 0)
+		// Runs Preprocess Workload to prepare the model
+		Inst().AppList = []string{"ml-workload-preprocess-rwx"}
```

Comment:
you need to take existing app list and revert it once done else remaining tests will also use same app

Reply:
Done